specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -16378,6 +16433,9 @@
           "description": "Resource tags"
         }
       },
```

Comment:
This model seems very different from the model seen in previous compute.json [files ](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json#L12141).
Why is it different?
Was Location removed as a required property at some point? When and why?

Reply:
it was removed in the earlier commits in this feature branch. we are going with different design so adding it back